examples/src/routes.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Cognite AS
+ */
+
+import React, { ReactNode } from "react";
+import { Simple } from "./pages/Simple";
+import { DistanceMeasurement } from "./pages/DistanceMeasurement";
+import { SideBySide } from "./pages/SideBySide";
+import { Clipping } from "./pages/Clipping";
+import { Filtering } from "./pages/Filtering";
+import { GpuSectorCuller } from "./pages/GpuSectorCuller";
+import { Migration } from "./pages/Migration";
+import { Picking } from "./pages/Picking";
+import { SectorWithPointcloud } from "./pages/SectorWithPointcloud";
+import { SimplePointcloud } from "./pages/SimplePointcloud";
+import { SSAO } from "./pages/SSAO";
+import { TwoModels } from "./pages/TwoModels";
+import { WalkablePath } from "./pages/WalkablePath";
+
+type ExampleRoute = {
+  path: string;
+  menuTitle: string;
+  component: ReactNode;
+};
+
+export const routes: Array<ExampleRoute> = [
+  {
+    path: "/simple",
+    menuTitle: "Simple",
+    component: <Simple />,
+  },
+  {
+    path: "/clipping",
+    menuTitle: "Clipping planes",
+    component: <Clipping />,
+  },
+  {
+    path: "/distance-measurement",
+    menuTitle: "Distance Measurement",
+    component: <DistanceMeasurement />,
+  },
+  {
+    path: "/filtering",
+    menuTitle: "Filtering",
+    component: <Filtering />,
+  },
+  {
+    path: "/gpu-sector-culler",
+    menuTitle: "GPU Sector Culler",
+    component: <GpuSectorCuller />,
+  },
+  {
+    path: "/migration?project=3ddemo",
+    menuTitle: "Migration",
+    component: <Migration />,
+  },
+  {
+    path: "/picking",
+    menuTitle: "Picking",
+    component: <Picking />,
+  },
+  {
+    path:
+      "/sector-with-pointcloud?project=3ddemo&model=5254077049582015&pointCloud=8182189276138640",
+    menuTitle: "Sector With Pointcloud",
+    component: <SectorWithPointcloud />,
+  },
+  {
+    path: "/side-by-side?project=3ddemo&model2=5254077049582015",
+    menuTitle: "Side-by-side debugger for sector models",
+    component: <SideBySide />,
+  },
+  {
+    path: "/simple-point-cloud?project=3ddemo&model=8182189276138640",
+    menuTitle: "Simple Point Cloud",
+    component: <SimplePointcloud />,
+  },
+  {
+    path: "/ssao",
+    menuTitle: "Screen space ambient occlusion shading",
+    component: <SSAO />,
+  },
+  {
+    // not really good defaults, provide something more meaningful
+    path:
+      "/two-models?project=3ddemo&model=1937792320363609&model2=5254077049582015",
```

Comment:
the default project is publicdata and we should not provide defaults for parameters like model